doc/nrf/includes/matter/introduction/no_sleep_thread_ftd.txt

```diff
@@ -0,0 +1,4 @@
+This device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power 802.15.4 Thread network as a Thread :ref:`Full Thread Device <thread_ot_device_types>`.
```

Comment:
```suggestion
This device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power 802.15.4 Thread network as a :ref:`Full Thread Device <thread_ot_device_types>`.
```